internal/hardware/hardware.go

```diff
@@ -89,3 +89,32 @@ func MapHardware(hardware *models.HardwareInfo) *v1alpha1.Hardware {
 	}
 	return &hw
 }
+
+func MapLabels(hardware *models.HardwareInfo) map[string]string {
+	if hardware == nil {
+		return nil
+	}
+	var labels = make(map[string]string)
+
+	for _, g := range hardware.Gpus {
```

Comment:
GPU information is no longer sent after https://github.com/jakub-dzon/k4e-device-worker/pull/73.

Reply:
OK, will remove